mineinabyss-features/src/main/kotlin/com/mineinabyss/anticheese/AntiCheeseListener.kt

```diff
@@ -36,4 +39,12 @@ class AntiCheeseListener: Listener {
             }
         }
     }
+
+    // Cancels pistons if a player is riding it via a GSit Seat
+    @EventHandler
+    fun BlockPistonExtendEvent.seatMovedByPiston() {
+        if (!AbyssContext.isGSitLoaded) return
```

Comment:
Make a new listener for the anticheese feature and only add the listener if GSit is loaded. This way our code doesnt need to be called every time the event gets fired even when it will never work anyways.